bin/propolis-server/src/lib/migrate/mod.rs

```diff
@@ -377,10 +377,8 @@ pub async fn dest_initiate(
 
     // TODO: https
     // TODO: We need to make sure the src_addr is a valid target
-    let src_migrate_url = format!(
-        "http://{}/instance/{}/migrate/start",
-        migrate_info.src_addr, migrate_info.src_uuid
-    );
+    let src_migrate_url =
+        format!("http://{}/instance/migrate/start", migrate_info.src_addr);
```

Comment:
I realize this was in the context of a formatting change to remove the instance ID, but do we want to hardcode http here? Perhaps it's something to address in a followup; maybe file a bug to audit protocol usage?

Reply:
There's a TODO for this nearby. I've filed #165 to track it.